lib/demos/material/data_table_demo.dart

```diff
@@ -67,13 +65,13 @@ class _DataTableDemoState extends State<DataTableDemo> with RestorationMixin {
       RestorableInt(PaginatedDataTable.defaultRowsPerPage);
   final RestorableBool _sortAscending = RestorableBool(true);
   final RestorableIntN _sortColumnIndex = RestorableIntN(null);
-  _DessertDataSource _dessertsDataSource;
+  _DessertDataSource? _dessertsDataSource;
```

Comment:
make non-nullable and late?

Reply:
so currently(before migration) we are only assigning with fresh values only if it's nonnull, making late would make it reassign every time in didChangeDependencies and restoreState, so should I do it ?

Reply:
Ah gotcha, then I would leave as is